### ISSUE TITLE
Add Flex/Bison syntax highlighting

### ISSUE THREAD

**Issue by Grandclosing** [notepad-plus-plus/notepad-plus-plus#4003]
I've scoured the internet to find such a thing, but it seems there hasn't be a Flex/Bison syntax highlighter plugin developed for Notepad++ yet. I'm not familiar at all with the process, so I'm wondering if anyone would like to take on such a thing, it'd be useful!

**Comment by GitMensch** [comment]
It would be indeed very useful but ideally should be a configurable multi-highlighter (you can have both tools generate C or C++ source).

**Comment by master-toma** [comment]
Please, add support for it!

**Comment by rafa-leo** [comment]
yeah, that a nice idea. Even VIM has a highlight for it. Add it please.
